docs/GettingStarted.md

```diff
@@ -16,14 +16,18 @@ npm install --save-dev jest
 Let's get started by writing a test for a hypothetical function that adds two numbers. First, create a `sum.js` file:
 
 ```javascript
-module.exports = (a, b) => a + b;
+function sum(a, b) {
```

Comment:
Can you update the [example](https://github.com/facebook/jest/blob/new-docs/examples/getting_started/sum.js) as well? The example actually used to define an actual `sum` function like you are using here, but I recently updated it to match this doc.